packages/lodestar-cli/tsconfig.json

```diff
@@ -4,7 +4,7 @@
   "compilerOptions": {
     "typeRoots": [
       "../../node_modules/@types",
-      "../../node_modules/@polkadot/ts"
```

Comment:
probably we want a separate module, or just fork polkadot-ts?

Reply:
I think we should fork polkadot-ts and rename